doc/examples/registration/plot_stitching.py

```diff
@@ -0,0 +1,174 @@
+===========================================
+
+This example demonstrates how a set of images can be assembled under
+the hypothesis of rigid body motions.
+
+"""
+
+from matplotlib import pyplot as plt
+import numpy as np
+from skimage import data, util, transform, feature, measure, filters, metrics
+
+
+def match_locations(img0, img1, coords0, coords1, radius=5, sigma=3):
+    """Match image locations using SSD minimization.
+
+    Areas from `img0` are matched with areas from `img1`. These areas
+    are defined as patches located around pixels with Gaussian
+    weights.
+
+    Parameters:
+    -----------
+    img0, img1 : 2D array
+        Input images.
+    coords0 : (2, m) array_like
+        Centers of the reference patches in `img0`.
+    coords1 : (2, n) array_like
+        Centers of the candidate patches in `img1`.
+    radius : int
+        Radius of the considered patches.
+    sigma : float
+        Standard deviation of the Gaussian kernel centered over the patches.
+
+    Returns:
+    --------
+    match_coords: (2, m) array
+        The points in `coords1` that are the closest corresponding matches to
+        those in `coords0` as determined by the (Gaussian weighted) sum of
+        squared differences between patches surrounding each point.
+    """
+    y, x = np.mgrid[-radius:radius + 1, -radius:radius + 1]
+    weights = np.exp(-0.5 * (x ** 2 + y ** 2) / sigma ** 2)
+    weights /= 2 * np.pi * sigma * sigma
+
+    match_list = []
+    for r0, c0 in coords0:
+        roi0 = img0[r0 - radius:r0 + radius + 1, c0 - radius:c0 + radius + 1]
+        roi1_list = [img1[r1 - radius:r1 + radius + 1,
+                          c1 - radius:c1 + radius + 1] for r1, c1 in coords1]
+        # sum of squared differences
+        ssd_list = [np.sum(weights * (roi0 - roi1) ** 2) for roi1 in roi1_list]
+        match_list.append(coords1[np.argmin(ssd_list)])
+
+    return np.array(match_list)
+
+
+############################################################################
+# Data generation
+# ---------------
+#
+# For this example, we generate a list of slightly tilted noisy images.
+
+img = data.moon()
+
+angle_list = [0, 5, 6, -2, 3, -4]
+center_list = [(0, 0), (10, 10), (5, 12), (11, 21), (21, 17), (43, 15)]
+
+img_list = [transform.rotate(img, angle=a, center=c)[40:240, 50:350]
+            for a, c in zip(angle_list, center_list)]
+ref_img = img_list[0].copy()
+
+img_list = [util.random_noise(filters.gaussian(im, 1.1), var=5e-4, seed=seed)
+            for seed, im in enumerate(img_list)]
+
+psnr_ref = metrics.peak_signal_noise_ratio(ref_img, img_list[0])
+
+############################################################################
+# Image registration
+# ------------------
+#
+# .. note::
+#    This step is performed using the approach described in the
+#    **Robust matching using Ransac** gallery example, but any other
+#    method from the **Image registration** section can be applied,
```

Comment:
Here I'm not sure how the Sphinx link to the #image-registration anchor should be done... Maybe
```suggestion
#    method from the :ref:`sphx_glr_auto_examples_registration` section can be applied,
```
?

Reply:
I confess that I don't really know how to properly manage Sphinx hyperlinks (that's why I didn't included them :sweat_smile:)

Reply:
All good, I'll take a look locally when I get a chance!